pvacseq/lib/net_chop.py

```diff
@@ -91,19 +91,23 @@ def main(args_input = sys.argv[1:]):
             score = -1
             pos = 0
             sequence_name = False
+            cleavage_scores = {}
             for line in results[i].split('\n'):
                 data = [word for word in line.strip().split(' ') if len(word)]
                 currentPosition = data[0]
+                isCleavage = data[2]
+                if isCleavage is not 'S':
+                    continue
                 currentScore = float(data[3])
                 if not sequence_name:
                     sequence_name = data[4]
-                if currentScore > score:
-                    score = currentScore
-                    pos = currentPosition
+                cleavage_scores[currentPosition] = currentScore
+            sorted_cleavage_scores = sorted(cleavage_scores.items(), key=lambda x: x[1], reverse=True)
```

Comment:
It may make more sense to have the cleavage scores sorted by position, rather than score, making it easier to see where in the peptide it's being cleaved (which is probably more important than the ranking of scores).  Anyone else have thoughts? @jhundal ?